### DISCUSSION TITLE
Query regarding translation.

### DISCUSSION THREAD

**Discussion by Nitin-789** [SAP/fundamental-ngx#11679] [Q&A]
Hi Everyone,
We have used a fdp-table and the strings which are internal to the component such as ("search" string in the search bar, "Ascending order" and "Descending order" options which are displayed when sorting in enabled on the table). I want them to be translated when I change the language in that same page which is another drop down. I checked the source code and I see that a custom pipe "fdTranslate" has been used to translate the strings. So is there any way by which I can change the strings which are internal to the table component when I change the language in the drop down ?

I'm using angular core version as "@angular/core": "15.2.0".

It would be great if you can guide me on how to approach this as I do not see any documentation(or may be I missed) related to it.

PS: Attaching few ss for the strings which I'm referring to.

Thanks,
Nitin

<img width="175" alt="Screenshot 2024-04-02 at 10 44 41 PM" src="https://github.com/SAP/fundamental-ngx/assets/37510228/a36597d3-3433-470b-8a93-c954dca60675">

<img width="456" alt="Screenshot 2024-04-02 at 10 44 55 PM" src="https://github.com/SAP/fundamental-ngx/assets/37510228/5823bf53-9034-43c1-ac13-9ac5f4b3f391">

**Comment by droshev** [comment]
@Nitin-789 We have those strings translated in version `0.43.32`. You can explore it [here](https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic)
<img width="457" alt="Screenshot 2024-04-03 at 12 38 00" src="https://github.com/SAP/fundamental-ngx/assets/4380815/39a756e7-1b42-4e36-aa05-caf71b43f543">
